_posts/2021-06-23-cockpit-247.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Cockpit 247
+author: kkoukiou
+date: 2021-06-23
+tags: cockpit linux
+slug: cockpit-247
+category: release
+summary: Cockpit with Metrics and Virtual Machines improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 247 and Cockpit Machines version 246.
+
+## Metrics: Grafana client setup
+
+The "Metrics settings" button introduced in [Cockpit 245](https://cockpit-project.org/blog/cockpit-245.html)
+has a new switch to set up exporting metrics to the network through the [PCP pmproxy service](https://linux.die.net/man/1/pmproxy)
+and the [Redis database](https://grafana-pcp.readthedocs.io/en/latest/datasources/redis.html):
+
+![metrics settings](/images/metrics-settings-2.png)
+
+If the machine uses firewalld, you can allow pmproxy in a particular zone afterwards:
+
+![metrics firewall zone selector](/images/metrics-fw-sel.png)
+
+After that, a remote [Grafana](https://grafana.com/) instance can read the machine's metrics through the
+[PCP Grafana plugin](https://grafana-pcp.readthedocs.io/), as described on the [PCP metrics feature page](https://cockpit-project.org/guide/latest/feature-pcp.html).
+
+## Machines: Share host files with the guest
+
+The user can now share a host directory tree with a guest. This is internally using [Virtio-fs](https://virtio-fs.gitlab.io/),
+which is designed to offer local file system semantics and performance.
+Virtio-fs is available in mainline since Linux 5.4, QEMU 5.0, libvirt 6.2 and Web console will not expose this feature in the UI if the requirements are not met.
+
+![shared directories card](/images/machines-shared-directories-card.png)
+
+![machines new shared directory dialog](/images/machines-new-shared-directory-dialog.png)
+
+## Machines: list USB, PCI and SCSI host devices that are passed through to the guest VM
```

Comment:
That's a bit too long.  Maybe "Machines: add list of pass-through devices"?

Reply:
Can it be 'Show list of ... '?